code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -352,7 +352,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/sniper_rifle
 	name = "sniper rifle"
-	desc = "The kind of gun that will leave you crying for mummy before you even realise your leg's missing"

```

Comment:
I did, nobody gave a shit (least of all me descriptions are always shit), you missed your chance to complain tbh.